server/routes/profile/profileStrengthsFunctions.ts

```diff
@@ -0,0 +1,30 @@
+import type { AlnScreenerList, AlnScreenerResponseDto, StrengthResponseDto, StrengthsList } from 'dto'
```

Comment:
I imagine when we do the "Challenges on the Overview page" story we'll create a similar file `profileChallengesFunctions.ts` for the exact same reason